app/templates/components/application-header.hbs

```diff
@@ -40,14 +40,14 @@
               <menu.item>
                 <span  class="dropdown-header">
                   Role:
-                  {{currentUser.roleName}}
+                  {{this.currentUser.roleName}}
                 </span>
               </menu.item>
               <menu.item>
-                <DynamicLink @route="{{settings-route settings.route}}" @model="{{settings.model}}">
+                <LinkTo @route="{{settings-route this.settings.route}}">
                   <i class="fas fa-cog"></i>
-                  Settings
-                </DynamicLink>
+                   Settings
+                </LinkTo>
```

Comment:
Just checking—does the link change correctly if you log in and then out with another account? I can't think of another reason why it would have been used.

Reply:
Good question.  I will check.  I think it works thanks to the settings-route, but will check.

Reply:
@codycooperross - there was a problem in determining the path for different users - supplying the 'dynamic' portion of the path.  It is fixed.

Reply:
Nice fix!